exchange/utils.go

```diff
@@ -700,3 +701,39 @@ func getExtBidAdjustmentFactors(requestExt *openrtb_ext.ExtRequest) map[string]f
 	}
 	return bidAdjustmentFactors
 }
+
+func buildXPrebidHeader(bidRequest *openrtb2.BidRequest, version string) string {
+	req := &openrtb_ext.RequestWrapper{BidRequest: bidRequest}
+
+	sb := new(strings.Builder)
```

Comment:
Syntax Nitpick: It's not common in Go to use the new keyword. Please consider using `sb := &strings.Builder{}`.